ci-operator/config/stolostron/search-indexer/stolostron-search-indexer-main.yaml

```diff
@@ -0,0 +1,79 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.17-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.17-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.17-linux
+  to: search-indexer
+promotion:
+  name: "2.5"
+  namespace: stolostron
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-indexer
+    env:
+      IMAGE_REPO: search-indexer
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: backplane-fast-forward
```

Comment:
Your fast-forwards are referencing backplane... these components are going into the main ACM, though, aren't they?

Reply:
No we aren't a backplane component at the moment.
I believe this is just the name for the prow job, we copied it blindly from another project. 

@mqhackett Please change it to `release-fast-forward` to avoid confusion.

Reply:
Yes, these are going into standard acm. However, when initially onbording to prow, the regular fast-forward job wasn't working so we used the backplane-fast-forward job